doc/source/io.rst

```diff
@@ -119,6 +119,8 @@ but all the metadata will be loaded. The *lazy_shape* attribute is added to all
 In this case, *lazy_shape* is a tuple that has the same value as *shape* with ``lazy=False``.
 To know if a class supports lazy mode use ``ClassIO.support_lazy``.
 By default (if not specified), ``lazy=False``, i.e. all data is loaded.
+The lazy option will be removed in future neo versions. Similar functionality will be
+implemented using ProxyObjects.
```

Comment:
since there is no `ProxyObject` class in Neo 0.6, we should say "proxy objects".